src/pages/error/ErrorPage.tsx

```diff
@@ -19,21 +19,22 @@ export const ErrorPage = ({ title, messages, backPath, status }: ErrorPageState)
   const {
     title: t,
     messages: m,
-    backPath: bp
+    backPath: bp,
+    status: s
   } = (state as ErrorPageState) || {
     title: 'This is a blank error page',
     messages: [], //['You might have found yourself here by reloading the error page or copying a link from somewhere else.'],
     backPath: '/'
   }
 
   useEffect(() => {
-    if (status === 401) {
+    if (status === 401 || s === 401) {
       toast({ title: '401 Nem vagy bejelentkezve', status: 'error' })
```

Comment:
Omit 401 here imo, the user does not care about the status code

Reply:
https://github.com/kir-dev/konzisite-frontend/issues/82